torch/nn/modules/adaptive.py

```diff
@@ -167,6 +167,14 @@ def reset_parameters(self) -> None:
             h2o.reset_parameters()
 
     def forward(self, input: Tensor, target: Tensor) -> _ASMoutput:
+        have_batch_dim = input.size(0) != self.in_features
+        if not have_batch_dim:
+            if target.dim() != 0:
+                raise RuntimeError('Target must be a scalar when there is no '
+                                   'batch dimension')
+            input = input.unsqueeze(0)
+            target = target.unsqueeze(0)
+
```

Comment:
Was this change intended for this PR? Looks like this is in `AdaptiveLogSoftmaxWithLoss`

Reply:
Yea this was a mistake. (committed to the wrong branch)